Google Play-services frequently crashes

This has been happening for about 2 months at this point, at least 1x per day, sometimes more. I can't think of any pattern, sometimes it happens when I use the phone and sometimes not.
```
type: crash
osVersion: google/caiman/caiman:16/BP4A.260205.002/2026021201:user/release-keys
package: com.google.android.gms:255034035, targetSdk 36
process: com.google.android.gms
processUptime: 18034639 + 587 ms
installer: app.grapheneos.apps
GmsCompatConfig version: 167

guea: rethrowing exception from lowpool[6]
	at bedq.run(:com.google.android.gms@255034035@25.50.34 (260400-855735097):43)
	at android.os.Handler.handleCallback(Handler.java:1070)
	at android.os.Handler.dispatchMessage(Handler.java:125)
	at cqvy.mv(:com.google.android.gms@255034035@25.50.34 (260400-855735097):1)
	at cqvy.dispatchMessage(:com.google.android.gms@255034035@25.50.34 (260400-855735097):140)
	at android.os.Looper.dispatchMessage(Looper.java:333)
	at android.os.Looper.loopOnce(Looper.java:263)
	at android.os.Looper.loop(Looper.java:367)
	at android.app.ActivityThread.main(ActivityThread.java:9331)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:566)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:837)
	Suppressed: glmn: 
		at tk_trace.ScheduledTaskImpl(Unknown Source:0)
		at tk_trace.people-CallingCardCleanupTask_onRunTaskAsync(Unknown Source:0)
Caused by: android.database.CursorIndexOutOfBoundsException: Index -1 requested, with a size of 0
	at android.database.AbstractCursor.checkPosition(AbstractCursor.java:535)
	at android.database.AbstractWindowedCursor.checkPosition(AbstractWindowedCursor.java:139)
	at android.database.AbstractWindowedCursor.getInt(AbstractWindowedCursor.java:71)
	at android.database.CursorWrapper.getInt(CursorWrapper.java:127)
	at emlb.b(:com.google.android.gms@255034035@25.50.34 (260400-855735097):35)
	at emlb.a(:com.google.android.gms@255034035@25.50.34 (260400-855735097):385)
	at emla.b(:com.google.android.gms@255034035@25.50.34 (260400-855735097):35)
	at com.google.android.gms.people.service.cpg.callingcard.CallingCardCleanupTask.a(:com.google.android.gms@255034035@25.50.34 (260400-855735097):38)
	at cqxx.call(:com.google.android.gms@255034035@25.50.34 (260400-855735097):30)
	at java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at beeh.c(:com.google.android.gms@255034035@25.50.34 (260400-855735097):50)
	at beeh.run(:com.google.android.gms@255034035@25.50.34 (260400-855735097):73)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1154)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:652)
	at bejy.run(:com.google.android.gms@255034035@25.50.34 (260400-855735097):8)
	at java.lang.Thread.run(Thread.java:1563)
```

--- a/src/com/android/providers/contacts/ScopedContactsProvider.java
+++ b/src/com/android/providers/contacts/ScopedContactsProvider.java
@@ -81,6 +81,11 @@ public class ScopedContactsProvider extends RedirectedContentProvider {
         checkAccess(callerPackageState);
 
         if (ContactScopesStorage.isEmpty(callerPackageState)) {
+            if (projection != null && projection.length == 1 && BaseColumns._COUNT.equals(projection[0])) {
+                var res = new MatrixCursor(projection, 1);
+                res.addRow(new Object[] { Integer.valueOf(0) });
+                return res;
+            }
             return ContactsProvider2.createEmptyCursor(uri, projection, false);
         }
 

